.script/workbooksValidations/previewImageValidator.ts

```diff
@@ -0,0 +1,33 @@
+import { WorkbookValidationError } from "../utils/ValidationError";
+import { WorkbookMetadata } from "../utils/WorkbookMetadata";
+
+const previewImageRegex: RegExp = /(black|white|Black|White)(\d*?).png/;
+const blackPreviewImageRegex: RegExp = /(black|Black)(\d*?).png/;
+const whitePreviewImageRegex: RegExp = /(white|White)(\d*?).png/;
+
+function isAllValidFileNames(previewImagesFileNames: Array<string>): boolean {
+    return previewImagesFileNames.every((previewImageFileName: string) => previewImageRegex.test(previewImageFileName));
```

Comment:
Couldn't you just check for matching the white or black regex here instead of adding a third one?
```
(previewImageFileName: string) => blackPreviewImageRegex.test(previewImageFileName) | whitePreviewImageRegex.test(previewImageFileName)
```

Reply:
definitely. however I decided to get rid of the RegExp entirely because the `includes` method actually is good enough for this requirement (see other comment).